x-pack/plugins/security_solution/cypress/tasks/login.ts

```diff
@@ -290,7 +290,7 @@ export const getEnvAuth = (): User => {
  * It prevents tour to appear during tests and cover UI elements
  * @param window - browser's window object
  */
-const disableRulesFeatureTour = (window: Window) => {
+const disableFeatureTourForRuleManagementPage = (window: Window) => {
```

Comment:
cc @MadameSheema -- more incoming cypress changes! 🙂

Reply:
Lots of thanks @spong!! This is really helpful :)